charts/validator/values.yaml

```diff
@@ -152,9 +136,9 @@ ghost:
     annotations: {}
       # external-dns.alpha.kubernetes.io/hostname: DOMAIN_NAME
 
-  # -- Ingress for the validator (Do not enable ingress, as libp2p does not support path based routing yet)
+  # -- Ingress for the validator - we should start using it maybe???
```

Comment:
We can test it out. Default ingress is only port 80 and 443. Of we can get Libp2p to listen on those then we should be able to use